test-api/schema-paginated/Node.js

```diff
@@ -2,7 +2,8 @@ import { nodeDefinitions, fromGlobalId } from 'graphql-relay'
 
 import joinMonster from '../../src/index'
 const options = {
-  minify: process.env.MINIFY == 1
+  minify: process.env.MINIFY == 1,
+  aliasPrefix: process.env.ALIAS_PREFIX
```

Comment:
Excellent!, we also need this line in schema paginated query root file and we are good to go!